docs/user/openstack/install_upi.md

```diff
@@ -336,22 +354,35 @@ values:
 
 <!--- e2e-openstack-upi: INCLUDE START --->
 ```sh
-$ python -c 'import yaml
+$ python -c 'import os
+import yaml
+import re
+re_os_net_id = re.compile(r"{{\s*os_net_id\s*}}")
+os_net_id = os.getenv("OS_NET_ID")
+path = "common.yaml"
+facts = None
```

Comment:
We want `facts` to be a dict, so let's it be a dict from the start.

Reply:
It doesn't matter if it will be None or dict. Empty dict will serve the same purpose as None.